badge/index.d.ts

```diff
@@ -46,7 +46,7 @@ type BadgeOptions = {
    * Widget position. Default is `bottom-right`.
    */
   position?: 'top-left' | 'top-center' | 'top-right' |
-             'middle-left' | 'middle-center' | 'middle-right' |
+             'middle-left' | 'middle-center' | 'center-middle' | 'middle-right' |
```

Comment:
I decided to mark `center-middle` as bad practice.

`middle` is a center for vertical. `center` is a center to horizontal.

Reply:
But you left 'center-middle' in the code and in the tests. Why?